### PR TITLE
zaki/tooltip

### PR DESCRIPTION
**Pull request by zaki-hanafiah**
Changelog
---
- Added tooltip component.
- Added different alignments `top, bottom, left, right` for tooltip.
- Added tooltip to `Trading` app sidebar.
- Refactored `ClockHeader`.
- Added icon assets for sidebar components.
- Portfolio drawer and layout fixes after updating branch with latest binary-static changes.

`Tooltip` Usage
---
Suffix Information Icon:
`<Tooltip alignment='left' message={message} is_icon />`

Normal usage
`<Tooltip alignment='top' message={message}>Text Here</Tooltip>`

### CODE REVIEW

File: src/sass/app_2/trading.scss [hunk 1]
```diff
@@ -1,34 +1,71 @@
...
+    margin-bottom: 31px;
+
+    #trade_app {
+        overflow: hidden;
+
+        .notice-msg {
+            background: unset;
+            /* stylelint-disable color-no-hex */
+            border: 1px solid #f2f2f2;
+            /* stylelint-enable color-no-hex */
```
Comment (binary-hamedanchi) [inline src/sass/app_2/trading.scss]: It's possible to disable one line:
```css
border: 1px solid #f2f2f2; /* stylelint-disable-line color-no-hex */
```

File: src/sass/app_2/trading.scss [hunk 2]
```diff
@@ -1,34 +1,71 @@
...
+                                content: url('../images/trading_app/sidebar/light/ic_digit_light.svg');
+                            }
+                        }
+                    }
+                    &.right {
+                        top: -3px;
+                        right: 5px;
+                        /* stylelint-disable number-max-precision */
+                        font-size: 0.5625rem;
+                        /* stylelint-enable number-max-precision */
```
Comment (binary-hamedanchi) [inline src/sass/app_2/trading.scss]: Same here :)

File: src/templates/static/new_layout/style_guide.jsx [hunk 3]
```diff
@@ -92,6 +93,39 @@ const StyleGuide = () => (
...
                     </div>
                 </div>
+                <div className='gr-row gr-padding-20'>
+                    <div className='gr-12'>
+                        <h2 className='center-text'>Tooltips</h2>
+                        <div className='gr-row'>
+                            <div className='gr-6'>
+                                <p className='center-text'>
+                                    <Tooltip alignment='top' message='Helper text here.' />
+                                    <span style={{ fontSize: '12px' }}>Top</span>
```
Comment (binary-hamedanchi) [inline src/templates/static/new_layout/style_guide.jsx]: How can we avoid inline styles?

Comment (zakibinary) [reply src/templates/static/new_layout/style_guide.jsx]: I think we can ignore it for now in this page, as this style-guide is just temporary and we should replace it with a style-guide page that shows both code and preview (similar to binary-style) so we can also include static components which are stateful like `Tabs` and `SideMenu`.

Comment (binary-hamedanchi) [reply src/templates/static/new_layout/style_guide.jsx]: Sure, thanks.

File: src/javascript/app_2/pages/trading/components/elements/fieldset.jsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        let header_time;
+        if (this.props.time) {
+            header_time = moment(this.props.time || undefined).utc().format('YYYY-MM-DD HH:mm:ss [GMT]');
+        }
+        return (
+            <fieldset>
+                <div className='fieldset-header'>
+                    <span className={field_left_class}>{this.props.header}</span>
+                    <span className='field-info right'>
+                        {this.props.time ?
```
Comment (binary-hamedanchi) [inline src/javascript/app_2/pages/trading/components/elements/fieldset.jsx]: Could be simply `{header_time}` :)

**Pull request merged**